src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -5084,7 +5084,7 @@ internal enum ErrorCode
             ERR_ExpressionTreeContainsBadCoalesce = 845,
             ERR_ArrayInitializerExpected = 846,
             ERR_ArrayInitializerIncorrectLength = 847,
-            ERR_OverloadRefOutCtor = 851,
+            ERR_OverloadRefKindCtor = 851,
```

Comment:
Yes. Is that needed? Just refactoring the name to indicate it is used now for all ref kinds instead of ref/out.

---
In reply to: [104460669](https://github.com/dotnet/roslyn/pull/17547#discussion_r104460669) [](ancestors = 104460669)